runtime/guard-jwt/src/main/java/io/aklivity/zilla/runtime/guard/jwt/internal/JwtGuardHandler.java

```diff
@@ -244,6 +257,43 @@ private JwtSessionStore supplySessionStore(
         return sessionStoresByContextId.computeIfAbsent(contextId, JwtSessionStore::new);
     }
 
+    private Map<String, JsonWebKey> getKeys(List<JwtKeyConfig> keysConfig)
```

Comment:
Please check all method definitions and put parameters on separate newlines below method name.

Reply:
what do you mean by "check all method definitions"? Check if the `keysConfig` parameter is null?

Reply:
Sorry, I just meant formatting.

For each method, make sure it is of the form:
```
... <method-name>(
  <param-type> <param-name>,
  <param-type> <param-name>)
```

rather than on a single line. 😄